expression/expressions/helper.go

```diff
@@ -467,3 +475,45 @@ func EvalBoolExpr(ctx context.Context, expr expression.Expression, m map[interfa
 
 	return x != 0, nil
 }
+
+// CheckOneColumn checks whether expression e has only one column for the evaluation result.
+// Now most of the expressions have one column except Row expression.
+func CheckOneColumn(e expression.Expression) error {
+	_, ok := e.(*Row)
```

Comment:
no need, row(1) is valid. 

because later we may check subselect expression, not only row.